docs/reference/commandline/run.md

```diff
@@ -186,6 +186,64 @@ flag makes Docker attempt to create a new file and write the container ID to it.
 If the file exists already, Docker returns an error. Docker closes this
 file when `docker run` exits.
 
+### <a name="pid"></a> PID settings (--pid)
+
+```text
+--pid=""  : Set the PID (Process) Namespace mode for the container,
+             'container:<name|id>': joins another container's PID namespace
+             'host': use the host's PID namespace inside the container
+```
+
+By default, all containers have the PID namespace enabled.
+
+PID namespace provides separation of processes. The PID Namespace removes the
+view of the system processes, and allows process ids to be reused including
+PID 1.
+
+In certain cases you want your container to share the host's process namespace,
+basically allowing processes within the container to see all of the processes
+on the system. For example, you could build a container with debugging tools
+like `strace` or `gdb`, but want to use these tools when debugging processes
+within the container.
+
+#### Example: run htop inside a container
+
+Create this Dockerfile:
```

Comment:
For a follow-up; this example could use an introduction to outline what we're about to do (run `htop` inside a container, but the host's PID namespace, which shows processes on the host (not just in the container itself).

And perhaps we should show _both_ (once with `--pid=host`, once without, to show the difference)